src/cmd/init.rs

```diff
@@ -131,6 +205,21 @@ fn populate(path: &Path, compile_sass: bool, config: &str) -> Result<()> {
         create_dir(path.join("sass"))?;
     }
 
+    setup_default_theme(&path.join("themes"))?;
+    Ok(())
+}
+
+fn setup_default_theme(path: &Path) -> Result<()> {
```

Comment:
I think we don't want a theme, we just want some default templates. We do have some that get automatically used in `serve` and I would prefer expanding those. Once they are in a good shape, we can write them to disk in the init.

Reply:
I've tried to look for them but for the life of me I can't find where those get defined. Mind pointing me in the right direction?

Reply:
https://github.com/getzola/zola/blob/master/components/utils/src/templates.rs extremely basic as you can see, https://github.com/getzola/zola/blob/master/components/utils/src/default_tpl.html for the actual template.